scripts/mixins/families/hpemde.lua

```diff
@@ -32,18 +32,24 @@ local function surface(mob)
 end
 
 local function openMouth(mob)
-    mob:addMod(xi.mod.ATTP, 100)
-    mob:addMod(xi.mod.DEFP, -50)
-    mob:addMod(xi.mod.DMGMAGIC, -5000)
+    -- Double attack
+    mob:setMod(xi.mod.ATTP, 100)
```

Comment:
This is supposed to be Base Damage * 2

Under `uint16 GetWeaponDamage(CMobEntity* PMob, uint16 slot)` inside mobutils.cpp you can add the mod `MOBMOD_WEAPON_BONUS` to increase its base damage. It would be: `mob:setMobMod(xi.mobMod.WEAPON_BONUS, 200)`

I would add this and check if the weapon damage actually increases. I think the `GetWeaponDamage` might actually be called when the mob spawns. Would have to double check myself.